samples/amber.cc

```diff
@@ -337,9 +337,21 @@ class SampleDelegate : public amber::Delegate {
       return r;
 #endif  // AMBER_ENABLE_LODEPNG
 
-    // TODO(asuonpaa): Try to load a binary format.
+    // Try to load a binary format.
+    auto data = ReadFile(path_ + file_name);
+    if (data.empty())
+      return amber::Result("Failed to load buffer data " + file_name);
 
-    return amber::Result("Failed to load buffer data " + file_name);
+    for (auto d : data) {
+      amber::Value v;
+      v.SetIntValue(d);
+      buffer->values.push_back(v);
+    }
+
+    buffer->width = 1;
```

Comment:
Why are these set?

Reply:
That's the width and height a buffer gets when initializing it inline. I'm matching it to avoid errors in buffer comparison.